docs/changes.rst

```diff
@@ -150,7 +150,7 @@ Released on August 15th 2019.
 * Avoid sending multiple notifications for single event.
 * Add support for filtering changes.
 * Extended predefined periods for reporting.
-* Added support for Azure Repos webhooks.
+* Added support for Azure repository webhooks.
```

Comment:
The service is actually called "Azure Repos", see https://azure.microsoft.com/cs-cz/services/devops/repos/